### PR TITLE
Issue: 6 - fix readme and pr templates

### LINKED ISSUE #6: Added PR Template
Added Standard PR Template that is specially for docs and all

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR fix Issue no #6 
This PR contains

* Added Standard PR Template that is specially for docs and all 
* Added Issue templates

This PR template contains
## Checklist 

- [ ] The Guide has been linted against a language and grammar tool.
- [ ] The Guide is easy to follow up with and understand for new users.
- [ ] The Docsify build works locally without inducing any unwarranted changes.
It is specially use for docs and started projects

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @t-fine

**Review by joewxboy — COMMENTED** [review]
There is another PR with more detailed issue templates.  Can we keep the PR template in this PR but remove the issue template?

**Comment by Ayush7614** [comment]
> There is another PR with more detailed issue templates. Can we keep the PR template in this PR but remove the issue template?

Sure @joewxboy i make new PR for the issue template

**Comment by Ayush7614** [comment]
cc @joewxboy

**Comment by joewxboy** [comment]
@Ayush7614 Don't forget to link the PR to the issue that it's based on, and add `Issue: ### - ` to the beginning of the PR title.

**Comment by Ayush7614** [comment]
> @Ayush7614 Don't forget to link the PR to the issue that it's based on, and add `Issue-6: ` to the beginning of the PR title.

cc @joewxboy

**Comment by Ayush7614** [comment]
Cc @joewxboy

**Comment by Ayush7614** [comment]
cc @joewxboy @t-fine

**Comment by t-fine** [comment]
@Ayush7614 just FYI for future PRs, I believe what @joewxboy is asking for is to select Issue #6 in the `Development` section on the right, near the top (that will cause the issue to be closed when this PR is merged), and for the title of this PR to be in the format of how I edited it
Screen Shot 2022-03-07 at 5 23 29 PM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Fixes #ISSUE_NUMBER 
+-->
+
+Fixes #
+
+## Checklist 
+
+- [ ] The Guide has been linted against a language and grammar tool.
+- [ ] The Guide is easy to follow up with and understand for new users.
```
Comment (t-fine) [inline .github/PULL_REQUEST_TEMPLATE.md]: Just making sure, do you mean this "Guide" is easy to follow and understand for new users? Or that new users are following up with this guide afterwards? If it's the former, I suggest something like "The Guide is easy to follow and understand for new users."

**Review by t-fine — CHANGES_REQUESTED** [review]

**Comment by Ayush7614** [comment]
cc @t-fine @joewxboy

**Review by t-fine — APPROVED** [review]

**Review by joewxboy — COMMENTED** [review]
What is a `Docsify` build?

**Comment by Ayush7614** [comment]
> What is a `Docsify` build?

I removed that line we can use this line later when we write proper docs it is use for docs but currently the docs and all things are not prepared

**Comment by Ayush7614** [comment]
Cc @joewxboy

**Review by joewxboy — APPROVED** [review]

**Pull request merged**